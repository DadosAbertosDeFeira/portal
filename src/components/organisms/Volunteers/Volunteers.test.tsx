import { render } from "@testing-library/react";
import type { Volunteer } from "@utils/volunteers";

import { Volunteers, type VolunteersProps } from ".";

describe("Volunteers", () => {
  const volunteersMock: Volunteer[] = [
    {
      github: "alvarogfn",
      instagram: "alvarogfn",
      linkedin: "alvarogfn",
      name: "Alvaro Guimar√£es",
      picture: "/assets/volunteers/anapaulagomes.jpeg",
      role: "Desenvolvedor",
      twitter: "alvarogfn",
    },
  ];

  const makeSut = ({
    volunteers = volunteersMock,
    ...props
  }: Partial<VolunteersProps>) => {
    return render(<Volunteers volunteers={volunteers} {...props} />);
  };

  it("should renders correctly", () => {
    const { asFragment } = makeSut({});

    expect(asFragment).toMatchSnapshot();
  });
});
