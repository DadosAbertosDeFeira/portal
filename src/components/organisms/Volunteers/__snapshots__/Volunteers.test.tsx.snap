// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Volunteers should renders correctly 1`] = `[Function]`;
